.github/deploy.sh

```diff
@@ -33,12 +33,11 @@ cd out
 git config user.name "GHA CI"
 git config user.email "gha@ci.invalid"
 
-if git diff --exit-code --quiet; then
-  echo "No changes to the output on this push; exiting."
-  exit 0
-fi
-
 if [[ -n $TAG_NAME ]]; then
+  if git diff --exit-code --quiet -- $TAG_NAME/; then
```

Comment:
This would have introduced a new bug, since `$TAG_NAME/` only has untracked files, therefore, this would always exit and never push anything.

Reply:
At least for new tags. Do we need a check for recreating an existing tag? 🤔

Reply:
c65eb4d should fix this